notebooks/Data Source Comparison.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
I see a lot of familiar code in here. Did you use a tool to help you build this file?

Reply:
Oh this is a jupyter notebook! I'd like to move more this code out of the notebook - planning on doing some of that when I write the next notebook.